Basic-Car-Maintenance/Shared/Dashboard/ViewModels/DashboardViewModel.swift

```diff
@@ -10,17 +10,13 @@ import FirebaseFirestoreSwift
 import Foundation
 
 @Observable
-class DashboardViewModel {
-    
-    let authenticationViewModel: AuthenticationViewModel
-    
+class DashboardViewModel: ObservableObject {
```

Comment:
I don't think this should be conforming to `ObservableObject`, and using the `@Observable` macro